### ISSUE TITLE
Helmet - Logo 1 Issue

### ISSUE THREAD

**Issue by SAKamjin** [emperorshammer/issues-incoming-queue#10]
1) Import user 2758
2) Click Pilot's Helmet
3) Logo 1 - populates with an image and defaults to Image - Stencil mask
4) Select Image - bg. transparent
5) error message does not have transparent background
6) image is blank
7) click button ... next to blank field under image - bg. transparent under Logo 1
8) file dialogue box opens with all files under misc folder
9) user can select any file and receives no transparent background open
10) switch to image - stencil mask
11) file is blank
12) repeat step 7 (I selected moh.gif)
13) MOH stencil now is inserted onto the helmet

Question - should we fix it to default back to the logo options or should we relocate the other files such that it's clear what file the user is to select with this error.

**Comment by SmTaUcM** [comment]
Yeah an interesting one there, I also notice that in TTT2, using the default image 'implogo.gif', it does allow the selection of 'image - bg. transparent'. So I've somehow deviated there.

I agree that it's not very clear which pictures are allowed and aren't allow for each category with the current system.
Even more awkwardly, some images fall into both categories.

On the note of locking image to pre-defined ones, I shall leave that to the admirals to decide. TTT2 allows selection of any image file which is why it's included in TTT3.

**Comment by SAKamjin** [comment]
> I agree that it's not very clear which pictures are allowed and aren't allow for each category with the current system.
> Even more awkwardly, some images fall into both categories.

Ideally if the folder the user is directed to only has logos versus other files the situation would be temporarily solved.

**Comment by SmTaUcM** [comment]
Going through the steps you've listed, it is operating as I intended. But there's definitely room for improvement.
I do recall having some challenges with masking transparent backgrounds which thinking about it, is likely why TTT3 more strict in regards to the selection of images.

@turtlexiv what are your thoughts and feelings? Is this something you would like to address prior to release?
